apps/daimo-mobile/src/view/screen/AddDeviceScreen.tsx

```diff
@@ -113,8 +113,7 @@ export function AddDeviceScreen() {
         <>
           <TextCenter>
             <TextH2>
-              Scanned{" "}
-              <TextBold>Device {keySlotToDeviceIdentifier(nextSlot)}</TextBold>
+              Scanned <TextBold>{keySlotTokeyLabel(nextSlot)}</TextBold>
```

Comment:
do we need the UI diff in this PR?
small enough that it looks OK to me, but curious if there's a reason we need it

Reply:
made it a bit simpler for me to extract this, I went file by file selecting or unselecting files for this and that covers diff neatly. happy to drop if we're trying to minimise diff?